src/commands/synthetics/interfaces.ts

```diff
@@ -178,8 +178,13 @@ export interface ConfigOverride {
   variables?: {[key: string]: string}
 }
 
-export interface Payload extends ConfigOverride {
+export interface Payload {
   metadata?: Metadata
+  tests: TestPayload[]
+}
+
+export interface TestPayload extends ConfigOverride {
+  executionRule: ExecutionRule
```

Comment:
Since it is needed for all `skipped` tests (due to config override or CI options) it is easier to always send it, it doesn't change anything on the backend so we can keep it like this.